qa/os/src/test/java/org/elasticsearch/packaging/test/ArchiveTests.java

```diff
@@ -106,43 +102,13 @@ public void test31BadJavaHome() throws Exception {
 
     }
 
-    public void test40CreateKeystoreManually() throws Exception {
```

Comment:
The tests removed here have been moved to the KeystoreManagementTests test class.